frontend/src/stores/logs.store.ts

```diff
@@ -5,17 +5,21 @@ import { ref } from 'vue'
 export const useLogsStore = defineStore('logs', () => {
   // State
   const logs = ref<Log[]>([])
-  const limit = ref(100)
+  const total = ref(50)
```

Comment:
```suggestion
  const total = ref(0)
```

Reply:
I think 0 is not a good idea, if by any bad chance, the value from the backend cannot be parsed correctly, ther will be no display at all. So I suggest putting some arbitrary non 0 value (anyways it is overwritten by the first sync)

Reply:
but also no problem to change this value later it needed.